setup.py

```diff
@@ -13,7 +13,7 @@
     packages=find_packages(),
     install_requires=(
         'aldryn-addons',
-        'django-cms==4.0.0dev11',
+        'django-cms',
```

Comment:
```suggestion
        'django-cms==4.0.0dev11',
```

Reverse the removal of pinning for testing.